.gitignore

```diff
@@ -97,3 +97,5 @@ ENV/
 *~
 *.swo
 *.swp
+
+notebooks/mydask.png
```

Comment:
Hmm... now we don't have other. At some point we might, and some might be useful. Let's see.